pkg/client/fibre_channel_session_object_set.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Hewlett Packard Enterprise Development LP
+
+package client
+
+import (
+	"reflect"
+
+	"github.com/hpe-storage/common-host-libs/jsonutil"
+	"github.hpe.com/nimble-dcs/golang-sdk/pkg/client/v1/model"
+	"github.hpe.com/nimble-dcs/golang-sdk/pkg/util"
+)
+
+
+// Fibre Channel session is created when Fibre Channel initiator connects to this group.
+const (
+    fibreChannelSessionPath = "fibre_channel_sessions"
+)
+
+
+// FibreChannelSessionObjectSet
+type FibreChannelSessionObjectSet struct {
+    Client *GroupMgmtClient
+}
+
+// CreateObject creates a new FibreChannelSession object
+func (objectSet *FibreChannelSessionObjectSet) CreateObject(payload *model.FibreChannelSession) (*model.FibreChannelSession, error) {
+	response, err := objectSet.Client.Post(fibreChannelSessionPath, payload)
+	return response.(*model.FibreChannelSession), err
+}
+
+// UpdateObject Modify existing FibreChannelSession object
+func (objectSet *FibreChannelSessionObjectSet) UpdateObject(id string, payload *model.FibreChannelSession) (*model.FibreChannelSession, error) {
+	response, err := objectSet.Client.Put(fibreChannelSessionPath, id, payload)
+	return response.(*model.FibreChannelSession), err
+}
+
+// DeleteObject deletes the FibreChannelSession object with the specified ID
+func (objectSet *FibreChannelSessionObjectSet) DeleteObject(id string) error {
+	return objectSet.Client.Delete(fibreChannelSessionPath, id)
+}
+
+// GetObject returns a FibreChannelSession object with the given ID
+func (objectSet *FibreChannelSessionObjectSet) GetObject(id string) (*model.FibreChannelSession, error) {
+	response, err := objectSet.Client.Get(fibreChannelSessionPath, id, model.FibreChannelSession{})
+	if response == nil {
+		return nil, err
```

Comment:
nit: I think the tabs needs to be fixed throughout the code.

Reply:
This looks okay in the vs code. Looks like browser showing as if there is more space than a tab.